src/main/java/core/basesyntax/MatrixOperation.java

```diff
@@ -6,18 +6,48 @@
  */
 public class MatrixOperation {
     public int[] getDiagonal(int[][] matrix) {
-        return null;
+        int [] diagonal = new int [matrix.length];
+        for (int i = 0; i < matrix.length; i++) {
+            for (int j = 0; j < matrix.length; j++) {
```

Comment:
read checklist attentively https://mate-academy.github.io/jv-program-fulltime/01_core/checklist/hw_02.html
- Use only one for loop to find diagonals of matrix, because the matrix is square.